src/components/StudentGroupPreview.tsx

```diff
@@ -19,30 +21,31 @@ const ButtonDiv = styled.div`
   display: flex;
 `
 
-interface IClassroom {
-  studentGroupName: string
-  students: []
-  docId: string
-}
+// interface IClassroom {
+//   studentGroupName: string
+//   students: []
+//   docId: string
+// }
 
 type StudentGroupProps = {
-  studentGroupName: string
   studentGroupId: string
-  doc: IClassroom
+  studentGroupName: string
+  userId: string
 }
 
-const StudentGroup: React.FC<StudentGroupProps> = ({ doc }) => {
-  const studentGroupRef = useFirestore().collection('studentGroups')
+const StudentGroup: React.FC<StudentGroupProps> = ({ userId, studentGroupId, studentGroupName }) => {
+  const studentGroupRef = useFirestore().collection('teachers').doc(userId).collection('studentGroups')
+
+  const history = useHistory()
 
   const openStudentGroupHandler = () => {
-    console.log('hello')
+    history.push('/student-group/' + studentGroupId)
```

Comment:
I prefer the syntax of template literals for this kind of thing

`/student-group/${studentGroupId}`